packages/url_launcher/url_launcher_macos/macos/Classes/UrlLauncherPlugin.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should this be the the standard license header we use?

Reply:
Yeah, updated